lib/util/template/index.ts

```diff
@@ -122,7 +122,7 @@ export const allowedFields = {
   prettyNewMajor: 'The new major value with v prepended to it.',
   prettyNewVersion: 'The new version value with v prepended to it.',
   project: 'ChangeLogProject object',
-  recreateClosed: 'If true, this PR will be recreated if closed',
+  recreateClosed: 'If "always", this PR will be recreated if closed',
```

Comment:
this PR will be breaking if this field is used as bool inside templates. not sure if we can migrate them automatically.

Reply:
templates?

Reply:
you can use this config option in any template field, like commit message. 

@rarkins WDYT about the breaking behavior?

Reply:
Yes, confirmed it's here: https://docs.renovatebot.com/templates/

Even if in a breaking change, that doesn't help any hosted users / repo configs.

One possibility would be:
- Use a new name, instead of reusing the old
- Populate boolean `recreateClosed` in templates for backwards compatibility